package.json

```diff
@@ -10,8 +10,11 @@
     "blockchain": "ganache-cli --mnemonic \"indicate swing place chair flight used hammer soon photo region volume shuffle\"",
     "blockchain:contracts": "node --experimental-modules ./integration/local/setup-contracts.mjs",
     "build": "run-s check-types build:css build:app",
-    "build:css": "cross-env NODE_ENV=production postcss src/tailwind.css -o src/index.css",
-    "build:app": "cross-env NODE_ENV=production webpack --progress --mode production",
+    "build:dev": "cross-env NODE_ENV=development npm run build",
+    "build:test": "cross-env NODE_ENV=test npm run build",
+    "build:prod": "cross-env NODE_ENV=production NET=mainnet npm run build",
```

Comment:
help me understand the use of `build:dev` and `build:test`? so later at netlify end the build command u will update to `npm run build:prod` ah?

Reply:
Yea, the test one will be used for the github test. The dev one for dev site and the prod one for prod site in netlify. Somehow I had to do this so that the tests don't fail because the jest env uses test. And then now that dev and prod have their own set of networks, need to split again.😑